### PR TITLE
Display 10 default queries when there is no recently reviewed query

### PR DESCRIPTION
**Pull request by YannanGao-gs**
## Summary

Display 10 default queries when there is no recently reviewed query

## How did you test this change?

- [ ] Test(s) added
- [x] Manual testing (please provide screenshots/recordings)
- [ ] No testing (please provide an explanation)

### CODE REVIEW

File: packages/legend-application-query/src/stores/data-space/DataSpaceQueryCreatorStore.ts [hunk 1]
```diff
@@ -342,4 +348,29 @@ export class DataSpaceQueryCreatorStore extends QueryEditorStore {
       ),
     );
   }
+
```
Comment (MauricioUyaguari) [inline packages/legend-application-query/src/stores/data-space/DataSpaceQueryCreatorStore.ts:351]: what happens when user enters the default route of query (no project info). 
Do we have a search specification for this case ?

Comment (YannanGao-gs) [reply packages/legend-application-query/src/stores/data-space/DataSpaceQueryCreatorStore.ts:351]: by default,
the function is empty 

https://github.com/finos/legend-studio/pull/3212/files#diff-5dae4710ada9d402191df7256d5cc7d4502e3b40a8f1b9acf960143f7ee6adfeR437

Comment (MauricioUyaguari) [reply packages/legend-application-query/src/stores/data-space/DataSpaceQueryCreatorStore.ts:351]: yeah but what does that return in terms of queries ?

**Comment by MauricioUyaguari** [comment]
Does it make sensse to also run a query on queries owned by current user?

**Review by MauricioUyaguari — APPROVED** [review]

**Pull request merged**